GarbageCan.Web/Startup.cs

```diff
@@ -173,6 +175,51 @@ public void ConfigureServices(IServiceCollection services)
 
                 return client;
             });
+
+            services.AddSwaggerGen(c =>
+            {
+                c.EnableAnnotations();
+                c.SwaggerDoc("v1", new OpenApiInfo
+                {
+                    Title = "Garbage Can",
+                    Description = "REST api to interact with the Garbage Can Discord Bot",
+                    Version = "v1",
+                    Contact = new OpenApiContact
+                    {
+                        Name = "Joe Sorensen",
+                        Email = "no peeking",
+                        Url = new Uri("https://github.com/murl-digital")
+                    }
+                });
+            });
+
+            
+            var authConfig = Configuration.GetSection("Authentication").Get<AuthConfiguration>();
+            if (authConfig.Enabled)
+            {
+                services.AddCors(options =>
+                {
+                    options.AddPolicy(authConfig.Audience,
+                        builder => builder
+                            .AllowAnyOrigin()
+                            .AllowAnyMethod()
+                            .AllowAnyHeader()
+                            .Build()
+                    );
+                });
+
+                services.AddAuthentication(JwtBearerDefaults.AuthenticationScheme).AddJwtBearer(options =>
```

Comment:
this is going to be replaced by a different solution, it can be removed (see: [experiments/api-auth](https://github.com/murl-digital/garbage-can/tree/experiments/api-auth))

Reply:
Removed Authentication setup